src/sentry/projectoptions/defaults.py

```diff
@@ -166,5 +166,5 @@
 # The available loader SDK versions
 register(
     key="sentry:loader_available_sdk_versions",
-    epoch_defaults={1: ["latest", "7.x", "6.x", "5.x", "4.x"], 11: ["latest", "7.x"]},
+    epoch_defaults={1: ["7.x", "6.x", "5.x", "4.x"], 11: ["7.x"]},
```

Comment:
q (not blocking): What is 1 and 11 here?

Reply:
This is the epochs. We are currently at 13. All projects created at or after epoch 11 will only be able to select 7.x, before that they will be able to select 7, 6 ,5, 4. Don't remember the reasoning behind that but we likely don't want to confuse ppl by giving too many options.

Reply:
ahh I see. Didn't know that, thx!